### ISSUE TITLE
Not enable rest metadata service on xroad-security-server-standalone

### ISSUE THREAD

**Issue by yamatokataoka** [nordic-institute/X-Road#703]
In the xroad-security-server-standalone docker image, I couldn't consume rest metadata service. It seems to need to add this parameter to `PROXY_PARAMS` in `/etc/xroad/services/proxy.conf` to enable the rest metadata service.

```
-Dxroad.proxy.serverRestServiceHandlers=${SERVICE_REST_HANDLERS#?}"
```

could you check?

confirmed on `bionic-6.24.0` tag.

**Comment by petkivim** [comment]
Thanks for submitting a bug report. You're right, REST metaservices are not enabled by default in the Standalone Security Server Docker image with the `bionic-6.24.0` tag. 

You can manually enable REST metaservices by adding the below parameter to `PROXY_PARAMS` in `/etc/xroad/services/proxy.conf`:

```
-Dxroad.proxy.serverRestServiceHandlers=${SERVICE_REST_HANDLERS#?}"
```

After the edit `PROXY_PARAMS` should look like this:

```
PROXY_PARAMS=" -Xms100m -Xmx512m -XX:MaxMetaspaceSize=80m \
-Djavax.net.ssl.sessionCacheSize=10000 \
-Dlogback.configurationFile=/etc/xroad/conf.d/proxy-logback.xml \
-Dxroad.proxy.clientHandlers=${CLIENT_HANDLERS#?} \
-Dxroad.proxy.serverServiceHandlers=${SERVICE_HANDLERS#?} \
-Dxroad.proxy.serverRestServiceHandlers=${SERVICE_REST_HANDLERS#?}"
```

To  apply the change, you must restart the `xroad-proxy` process: `supervisorctl restart xroad-proxy`

The problem will be fixed in version `bionic-6.24.1` that will be released within a week.

**Comment by yamatokataoka** [comment]
I understood.
thank you for your quick reply.
